### PR TITLE
IT-5162 SelectListItemTile

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import SelectListItemTile from './SelectListItemTile.vue';
+import { TILE_STATES } from './../../Tile/Tile.consts';
+
+import { Args, Meta, StoryFn } from '@storybook/vue3';
+import { args, argTypes, data, template } from './../../Tile/Tile.sb.shared';
+
+export default {
+	title: 'Components/SelectListItemTile',
```
Comment (tweetgeek) [inline lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts:8]: why not in `SelectList` like others?
```suggestion
	title: 'Components/SelectList/SelectListItemTile',
```

File: lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	setup() {
+		return { ...args };
+	},
+	template: template('select-list-item-tile'),
+	data,
+});
+
+export const Static = StoryStaticTemplate.bind({});
+
+Static.argTypes = argTypes;
```
Comment (tweetgeek) [inline lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts:47]: why we use controls in Static template?

File: lib/js/components/SelectList/SelectListSectionTitle/SelectListSectionTitle.stories.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	setup() {
+		return { ...args };
+	},
+	template: `
+			<select-list>
+				<select-list-section-title :label="label" :is-uppercase="isUppercase" /> 
+			</select-list>`,
+});
+
+const args = {} as Args;
```
Comment (tweetgeek) [inline lib/js/components/SelectList/SelectListSectionTitle/SelectListSectionTitle.stories.ts:22]: it is unnecessary

**Review by tweetgeek — APPROVED** [review]

File: lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts (hunk 2 above)
Comment (pioziol123) [reply lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts:47]: hmm. I think I copied it without thinking :D

**Pull request merged**